examples/vision/bit.py

```diff
@@ -39,12 +40,14 @@
 ## Setup
 """
 
+import os
+os.environ["KERAS_BACKEND"] = "tensorflow"
 import numpy as np
 import pandas as pd
 import matplotlib.pyplot as plt
 
+import keras
 import tensorflow as tf
```

Comment:
@sitamgithub-MSIT all tf ops needs to be replaced with `keras.ops` and `import tensorflow as tf` should be removed. More details on migration here - https://keras.io/guides/migrating_to_keras_3/

Reply:
I read the guide. The only two places where tf is working in this example are: one is this ```AUTO = tf.data.AUTOTUNE```. Now, without using TF, I don't know how to do this with Keras. So, I need suggestions on that.

Another code block is this:
```
@tf.function
def preprocess_train(image, label):
    image = tf.image.random_flip_left_right(image)
    image = tf.image.resize(image, (RESIZE_TO, RESIZE_TO))
    image = tf.image.random_crop(image, (CROP_TO, CROP_TO, 3))
    image = image / 255.0
    return (image, label)


@tf.function
def preprocess_test(image, label):
    image = tf.image.resize(image, (RESIZE_TO, RESIZE_TO))
    image = image / 255.0
    return (image, label)
```
So, I made changes to the code as follows:
```
# @tf.function
def preprocess_train(image, label):

    image = random_flip(image)
    # image = tf.image.random_flip_left_right(image)
    image = keras.ops.image.resize(image, (RESIZE_TO, RESIZE_TO))
    # image = tf.image.random_crop(image, (CROP_TO, CROP_TO, 3))
    image = random_crop(image)
    image = image / 255.0
    return (image, label)


# @tf.function
def preprocess_test(image, label):
    image = keras.ops.image.resize(image, (RESIZE_TO, RESIZE_TO))
    image = keras.ops.cast(image, dtype="float32")
    image = image / 255.0
    return (image, label)
```
And this is working well with the overall code. So, the only necessity of tf is for ```tf.data.AUTOTUNE```. And I need help on that.